deerlab/utils/utils.py

```diff
@@ -3,9 +3,9 @@
 import numpy as np
 import scipy as scp
 import scipy.optimize as opt
-from types import FunctionType 
+from types import FunctionType, LambdaType
 from functools import wraps

```

Comment:
Needed?

Reply:
`LambdaType` is not needed. I removed it.